config/routes.rb

```diff
@@ -2,22 +2,6 @@
 
 Rails.application.routes.draw do
   get "legal/index"
-  
-  resources :market_places
-  root 'main#index'
-  # get 'main/dashboard'
-  resources :passwords, controller: "clearance/passwords", only: [:create, :new]
-  resource :session, controller: "clearance/sessions", only: [:create]
-
-  resources :users, only: [:create] do
-    resource :password,
-      controller: "clearance/passwords",
-      only: [:create, :edit, :update]
-  end
-
-  get "/sign_in" => "clearance/sessions#new", as: "sign_in"
-  delete "/sign_out" => "clearance/sessions#destroy", as: "sign_out"
-  get "/sign_up" => "clearance/users#new", as: "sign_up"
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
   root to: 'home#index'
   resources :home
```

Comment:
At the moment of pulling the master, it marks an error.

![imagen](https://user-images.githubusercontent.com/12243331/103812176-bb37b600-5023-11eb-8206-850d8632bbd3.png)